client/state/happychat/test/selectors.js

```diff
@@ -92,49 +92,33 @@ describe( 'selectors', () => {
 			HAPPYCHAT_CHAT_STATUS_NEW,
 		];
 
-		it( 'should return false if Happychat is unavailable', () => {
+		it( 'should return false if Happychat is not connected', () => {
```

Comment:
I know the `isAvailable` flag really means "Are operators available" but Maybe we should leave a test like "Should return true if operators are unavailable but currently chatting". Also I'm unsure if `connectionStatus==="connected"` is the same as "Is user currently chatting"